keystorage/aws/src/test/java/tech/pegasys/signers/aws/AwsSecretsManagerTest.java

```diff
@@ -265,9 +263,34 @@ private void closeClients() {
     closeTestSecretsManager();
   }
 
-  private String createSecret(final Tag tag, String secretValue)
+  private boolean checkTestSecretExists(
```

Comment:
It's a shame we have to spend two requests getting this info. I can see how it's useful to be able to update the secrets in storage just by changing the tests though. 

In the scenario where we update either the secret values or tags, I think this would leave orphaned keys...do you think we should delete any matching key names whose values/tags don't match as part of this too?